src/components/POLine/POLineDetails/FieldPaymentStatus.js

```diff
@@ -19,7 +19,7 @@ class FieldPaymentStatus extends Component {
       <Field
         component={Select}
         label={<FormattedMessage id="ui-orders.poLine.paymentStatus" />}
-        name="payment_status"
+        name="paymentStatus"
```

Comment:
Does this "name" correspond to field on line 29? If so, payment_status needs to be changed.

Reply:
No, they are independent, line 29 is a translation key and it's better to not touch this w/o a serious reason